### PR TITLE
Created a README file

### PR DESCRIPTION
**Pull request by MarkKrutik**
I have created the README file, please look at it and tell me if you need changes to be made. Also it would be good to create a contribution and license page on GitHub.
Thank you,

Mark Krutik

### CODE REVIEW

**Comment by MarkKrutik** [comment]
Also would you like me to upload the read me file to the master branch?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+Gopal Geol's Website
```
Comment (gopalgoel19) [inline README.md]: There is a typo. It is Gopal Goel.
Also, a better heading would be "Resume Website".

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+=
+
+The point of this web application is to give future managers and other programmers an idea of the skills,
+previously completed projects, education and test scores of a third-year engineering student.
```
Comment (gopalgoel19) [inline README.md]: instead of "third-year engineering student" use "software engineer"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+working on this interesting project. This website needs the help of as many programmers as possible. 
+Your help will ge greatly appreciated, and you will be awarded by hacktoberfest for your participation.
+
+Getting Started
+-
+This documentation will help you get started with participating in this open-source project.
+To participate in the project please fork by pressing the fork button on the top right and clone the 
+repository with this command:  
+        
+    git clone https://github.com/gopalgoel19/gopalgoel19.github.io.git
```
Comment (gopalgoel19) [inline README.md]: git clone https://github.com/{your-user-name}/gopalgoel19.github.io.git

File: README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+-
+Please open the index.html file and it will open the HTML file on the browser.
+
+Built with
+-
+* HTML5 - Is the latest version of a HyperText Markup Language that allows to describe web pages.  
+* CSS3 - Is a Style Sheet Language that allows to style an HTML5 document.  
+* JQuery - Is a JavaScript Library that simplifies how you program in JavaScript.  
+
+Authors:
```
Comment (gopalgoel19) [inline README.md]: We can skip Authors, IMO

**Comment by MarkKrutik** [comment]
Please let me know if this is good. I will then post this file to the master branch.

**Pull request merged**